alsid-syslog-proxy/AlsidForAD.log

```diff
@@ -0,0 +1,241 @@
+2020-10-20T15:22:23+00:00 julienc.alsid.app AlsidForAD[4]: "1" "1" "test-forest" "test-directory" "objectPath" "eventId" "eventType" "attributeName"="newValue"
```

Comment:
how is this file used?

Reply:
The file is used to provide a sample log file in the Readme, this file is useful to configure the custom log collection of a Log Analytic Workspace on the Azure portal.